exporter/awsxrayexporter/translator/cause.go

```diff
@@ -346,3 +348,84 @@ func fillPythonStacktrace(stacktrace string, exceptions []awsxray.Exception) []a
 
 	return exceptions
 }
+
+func fillJavaScriptStacktrace(stacktrace string, exceptions []awsxray.Exception) []awsxray.Exception {
+	r := textproto.NewReader(bufio.NewReader(strings.NewReader(stacktrace)))
+
+	// Skip first line containing top level exception / message
+	r.ReadLine()
+	exception := &exceptions[0]
+	var line string
+	line, err := r.ReadLine()
+	if err != nil {
+		return exceptions
+	}
+
+	exception.Stack = make([]awsxray.StackFrame, 0)
+	for {
+		if strings.HasPrefix(line, "\tat ") {
+			parenIdx := strings.IndexByte(line, '(')
+			if parenIdx >= 0 && line[len(line)-1] == ')' {
+				label := line[len("\tat "):parenIdx]
+				path := line[parenIdx+1 : len(line)-1]
+				line := 0
+
+				colonFirstIdx := strings.IndexByte(path, ':')
+				colonSecondIdx := indexOf(path, ':', colonFirstIdx)
+
+				if colonFirstIdx >= 0 && colonSecondIdx >= 0 && colonFirstIdx != colonSecondIdx {
+					lineStr := path[colonFirstIdx+1 : colonSecondIdx+1]
```

Comment:
`colonSecondIdx+1` seems weird here, doesn't this catch the colon into the slice?

Reply:
Ah, I accidentally compensated for the missing increment in my helper function, changing